gtk/CMakeLists.txt

```diff
@@ -50,13 +50,22 @@ add_custom_command(
 
 if(ENABLE_NLS)
     find_program(INTLTOOL_MERGE_EXECUTABLE intltool-merge REQUIRED)
+
     set(${PROJECT_NAME}_DESKTOP_FILE "${PROJECT_BINARY_DIR}/${TR_NAME}-gtk.desktop")
     add_custom_command(
         OUTPUT ${${PROJECT_NAME}_DESKTOP_FILE}
         COMMAND ${INTLTOOL_MERGE_EXECUTABLE} --desktop-style --utf8 ${CMAKE_SOURCE_DIR}/po ${PROJECT_SOURCE_DIR}/transmission-gtk.desktop.in ${${PROJECT_NAME}_DESKTOP_FILE}
         DEPENDS ${PROJECT_SOURCE_DIR}/transmission-gtk.desktop.in
         VERBATIM
     )
+
+    set(${PROJECT_NAME}_APPDATA_FILE "${PROJECT_BINARY_DIR}/${TR_NAME}-gtk.appdata.xml")
```

Comment:
I only added an appdata file for the GTK frontend because this is the one we'll promote in Endless OS. Of course, there's nothing stopping an appdata-and-Qt enthusiast from adding appdata for the Qt frontend.